MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2548,7 +2548,10 @@ public string GetHardwareAcceleratedVideoDecoder(VideoType videoType, MediaStrea
                 {
                     if (Environment.OSVersion.Platform == PlatformID.Win32NT)
                     {
-                        return "-hwaccel dxva2";
+                        if( Environment.OSVersion.Version.Major > 6 || ( Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor > 1 ) )
```

Comment:
```suggestion
                        if(Environment.OSVersion.Version.Major > 6 || (Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor > 1))
```

Reply:
The conditional above has a space after if - do we want that as the standard, or directly together like this? I can fix up one or the other.